### ISSUE TITLE
Add more entries to the parser

### ISSUE THREAD

**Issue by espenfl** [aiida-vasp/parsevasp#34]
This issue is a container for requests related to additions of new parser entries.

**Comment by espenfl** [comment]
There is a request to add other energy entries outside `energy_no_entropy`. https://github.com/aiida-vasp/parsevasp/issues/33#issue-636787564.

**Comment by zhubonan** [comment]
The other thing that would be nice is the runtime timings such as the total runtime, scf time and ionic steps times. The last two can be obtained using `grep LOOP OUTCAR` while the first one is only printed at the end.  

They can probably be used for estimating a suitable walltime limits for subsequent runs.

**Comment by zhubonan** [comment]
Hi @espenfl, I am trying to understand what the `energy_no_entropy` is.

I found the code here:

https://github.com/aiida-vasp/parsevasp/blob/44a39363e753438cfe3f0be77e9c4148ca290fca/parsevasp/vasprun.py#L1901-L1911

But it looks to me that it fetches the `e_0_energy` which is the extrapolated energy to 0 K (E-0.5TS). This is different from the `e_fr_energy` which is the energy that is minimised (free energy, E-TS), or `e_wo_entrp` which I think is the total energy (with fractional occupations). 

Does this mean that `energy_no_entropy` is in fact E-0.5TS?

**Comment by espenfl** [comment]
@zhubonan Indeed. Plan was to add the other energies, but did not get that far. Let us add all of those, so that the user can select. Default should still be `e_0_energy` (`energy(sigma->0)` in `OUTCAR`) as this is the extrapolated one that tries to remove the `sigma` smearing contributions etc. This is typically slightly higher than the `e_wo_entrp` and is what many use for the total energy in VASP. When running with different `sigma` and you want to compare, you have to look at this one, otherwise you might get tricked by the differences being caused by differences in `sigma` etc. And so forth. But we should anyway support the extraction of the others. Currently, that is not implemented, but should be easy to do. If you want, please give it a go.

**Comment by zhubonan** [comment]
I see, yeah `energy(sigma->0)` is the one we want. Just that the naming is a bit confusing. 
I will see if I can make it extract other energies as well. It could be useful when someone want to knwon how much the effect of SIGMA is having.

**Comment by lorenzovarrassi** [comment]
At the moment the parser does not fully support GW and response calculations. 
It would be nice to add support for G0W0 and optics calculations (LOPTICS - BSE).
